src/Microsoft.DotNet.Interactive.Http/HttpKernelExtension.cs

```diff
@@ -34,3 +35,23 @@ public class HttpKernelExtension
         }
     }
 }
+
+internal static class DateTimeOffsetExtensions
+{
+    public static DateTimeOffset AddOffset(this DateTimeOffset dateTime, int offset, string options)
```

Comment:
This extension method in this file is intended for use by consumers who load the http kernel via a .dib extension load 
script.

Could you please move this helper as a private (or nested) function inside HttpKernel itself?

Reply:
Alternatively, you could create a new file `DateTimeUtilities.cs` in the current project and move it there.

Reply:
(It is also generally preferred to have just a single type per C# code file in the current repo)